cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -299,7 +300,8 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
         """To not get an error creating the same index twice, we use the `DoNothingIfExists` operator for indexes.
           |This will make sure no other index with the given name or schema already exists before the `CreateIndex` operator creates an index on a property for all nodes having a certain label.
           |If it finds an index with the given name or schema it will stop the execution and no new index is created.
-          |The following query will create an index with the name `my_index` on the `name` property of nodes with the `Country` label only if no such index already exists.""".stripMargin,
+          |The following query will create an index with the name `my_index` on the `name` property of nodes with the `Country` label only if no such index already exists.
+          |Note: The `IF [NOT] EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.""".stripMargin,
```

Comment:
Since this is `CREATE` the `NOT` should not be optional
```suggestion
          |Note: The `IF NOT EXISTS` syntax for indexes is only available in Neo4j 4.1.3 and onwards.""".stripMargin,
```